src/data/services/lms/hooks/data.ts

```diff
@@ -44,14 +43,20 @@ export const usePageData = (): types.QueryData<types.PageData> => {
         ({ data }) => camelCaseObject(data)
       );
       */
-      return Promise.resolve(camelCaseObject(loadState({ view, progressKey })));
+      // const data = camelCaseObject(loadState({ view, progressKey }))
+      const data = loadState({ view, progressKey });
+      const result = {
+        ...camelCaseObject(data),
+      };
```

Comment:
why?

Reply:
needed to selective camelize earlier with selected option and forgot to change back.